src/index.js

```diff
@@ -119,21 +119,24 @@ export const updateGitlabRepoDependency = decorateFunctionLogger(({
 }) => {
     logger.trace(`time to clone and update gitlab repo ${repo}`);
     const cwd = `repos/gitlab/${repo}`;
+    const localBranch = getPackageBranchName(packageName);
     return Q.fcall(() => {
         logger.trace('clone');
         return exec(`git clone --depth 1 https://${gitlabUser}:${gitlabToken}@${gitlabHost}/${gitlabOrg}/${repo}.git ${cwd}`, { logger });
     }).then(() => {
         logger.trace('checkout');
-        return exec(`git checkout -B ${getPackageBranchName(packageName)}`, { cwd, logger });
+        return exec(`git checkout -B ${localBranch}`, { cwd, logger });
     }).then(() => {
         logger.trace('version bump');
         return updateDependency({
             path: path.resolve(cwd, 'package.json'),
             packageName,
             logger
         });
-    }).then(([before, after]) => (
-        Q.fcall(() => (
+    }).then(([before, after]) => {
+        const breakingChange = major(before) !== major(after);
+        const remoteBranch = `${breakingChange ? 'wip-' : ''}${localBranch}`;
```

Comment:
Do you want to put the major version in the branch name?  What happens if you have versions 3, 4, 5 at the same time, won't 4 & 5 collide?